checks/security_policy.go

```diff
@@ -48,5 +49,13 @@ func SecurityPolicy(c *checker.CheckRequest) checker.CheckResult {
 		c.RawResults.SecurityPolicyResults = rawData
 	}
 
-	return evaluation.SecurityPolicy(CheckSecurityPolicy, c.Dlogger, &rawData)
+	// Evaluate the probes.
+	findings, err := evaluateProbes(c, CheckSecurityPolicy, probes.SecurityPolicy)
```

Comment:
note: this expects `c.RawResults != nil`, and panics if that's not true. This seems like a loss in some functionality for someone calling  the function here at this top level. WDUT?

Note: Dependency-Update-Tool also has this problem, (and Packaging doesnt seem like it's structured the same as all the others)

Reply:
even if we add nil checks, we essentially throw away the raw results we did